driver/src/main/scala/api/AsyncDriver.scala

```diff
@@ -38,7 +38,6 @@ import com.typesafe.config.Config
  *
  * @param config a custom configuration (otherwise the default options are used)
  * @param classLoader a classloader used to load the actor system
- *
```

Comment:
revert

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370